composer.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "buildempire/apartment",
+  "description": "Multi-tenancy for Lumen using PostgreSQL schemas",
+  "type": "library",
+  "license": "MIT",
```

Comment:
Can we get a version number added in, so that we can use it for tagging releases please so that it all works as expected. :)